file-formats/intf/examples/z_aff_example_intf.intf.json

```diff
@@ -11,12 +11,6 @@
         "description": "This is an example type"
       }
     ],
-    "attributes": [
```

Comment:
Why is this happening again? Had some issues with this one this morning ...

Reply:
The description of the constant has been deleted. I add it and update the example.

Reply:
ah. @albertmink The deletion of the description happened because you renamed the constant to co_...   :-)